packages/react/src/patterns/sub-patterns/Card/__stories__/index.scss

```diff
@@ -5,5 +5,27 @@
 // LICENSE file in the root directory of this source tree.
 //
 
-@import '../../../../../../styles/scss/patterns/sub-patterns/card/index';
+@import '../../../../../../styles/scss/patterns/sub-patterns/card';
 @import '@carbon/grid/scss/index.scss';
+
+.#{$prefix}--card--white {
+  @include carbon--theme($carbon--theme--white) {
+    @include card;
+  }
+}
+.#{$prefix}--card--g10 {
+  @include carbon--theme($carbon--theme--g10) {
+    @include card;
+  }
+}
+.#{$prefix}--card--g90 {
+  @include carbon--theme($carbon--theme--g90) {
+    @include card;
+  }
+}
+
+.#{$prefix}--card--g100 {
+  @include carbon--theme($carbon--theme--g100) {
+    @include card;
+  }
+}
```

Comment:
Since this is all apart of just the story, could we change the background so that can be clearly seen?

Reply:
@photodow change the background?

Reply:
Right, `g10` for instance if you don't change the background of the story as well it just looks like some text sitting on a white background. If we could add a background to the story based on the theme I think it might go a long way helping to visualize it.

![image](https://user-images.githubusercontent.com/3793636/74752357-2aea5a00-5234-11ea-94d3-fdf258e0622e.png)

Reply:
done